src/helpers/determineLanguage.ts

```diff
@@ -0,0 +1,22 @@
+import { catalogs } from "config/i18n";
+
+// input could be in both "en" and "en-US" formats
+export function determineLanguage(input: string): string {
+  let language;
+
+  const [languageFromInput, countryFromInput] = input.split("-");
```

Comment:
You'll need to ensure that Vibrant is sending the country as well (I don't remember if it is).

- https://github.com/stellar/mobile-wallet/blob/master/src/components/AccountSetup/RecoverAccount/VerifyEmail.tsx#L52
- https://github.com/stellar/mobile-wallet/blob/master/src/components/AccountSetup/RecoverAccount/VerifyPhoneNumber.tsx#L28

Reply:
Yep, we still need to accommodate this change on the wallet. I was planning on doing it after we merge and deploy this PR . Thanks for the reminder!

Reply:
Since this is not a breaking change, it should be safe to update Vibrant at any point in time after the deployment